transport.rst

```diff
@@ -101,12 +101,43 @@ Matrix allows to check for the presence of a user. Clients should listen for cha
 Sending transfer messages to other nodes
 ----------------------------------------
 
+There are three different ways to send messages between Raiden nodes:
+
+* Messages in matrix chat rooms
+* Matrix ``toDevice`` messages
+* Messages over WebRTC channels
+
+Of those, only the matrix rooms based messages are a requirement, while the
+other options are pure optimizations that reduce latency.
+
+Matrix rooms
+''''''''''''
 Private channel rooms have random room_ids in the format ``!<random_room_id>:<homeserver-uri>``, where the ``random room_id`` is generated by the server.
 To avoid race conditions only the node with the lexicographically lower address will create rooms and invite peers.
 
 Since users may roam to other homeservers, participants should keep listening for user-join events in the discovery room.
 When a new user for an address of interest joins it should be invited into a communications room.
 
+Matrix ``toDevice`` messages
+''''''''''''''''''''''''''''
+Matrix also supports ``toDevice`` messages. These are not stored permanently as
+part of a shared communication history and are delivered exactly once to each
+client device.
+
+As Raiden does not rely on the messaging history, this is a valid alternative
+and usually shows lower latency and reduces load on the Matrix federation nodes.
+
+Nodes that support ``toDevice`` messages signal that with the ``toDevice``
```

Comment:
Maybe state that clients compatible with this capability are expected to have a `deviceId="RAIDEN"`, and clients sending this type of messages may specify `RAIDEN` as target deviceId.